tests/e2e/support/page/files/publicLink.ts

```diff
@@ -6,6 +6,7 @@ import util = require('util')
 
 export class PublicLink {
   private readonly actor: Actor
+  private static lastCreatedPublicLink: string
```

Comment:
@amrita-shrestha  you can put a commet why this is made static.

Reply:
sorry commet? Can't get what you mean?

Reply:
@amrita-shrestha  yes you can put a comment saying that if so !!